### PR TITLE
(google) Add support for subnetwork config default and gce_subnetwork…

### PR DESCRIPTION
**Pull request by duftler**
… extended attribute.

### CODE REVIEW

**Comment by duftler** [comment]
Closes https://github.com/spinnaker/spinnaker/issues/1391.

**Comment by duftler** [comment]
@lwander or @danielpeach please review.

File: rosco-web/config/packer/gce.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "gce_account_file": "",
     "gce_zone": null,
     "gce_network": null,
+    "gce_subnetwork": "",
```
Comment (lwander) [inline rosco-web/config/packer/gce.json]: Why is subnetwork initialized with `""` rather than `null` like the other `String` parameters above?

Comment (duftler) [inline rosco-web/config/packer/gce.json]: `null` means required, `""` means optional.

**Pull request merged**